openlibrary/catalog/get_ia.py

```diff
@@ -79,9 +82,13 @@ def get_marc_record_from_ia(identifier):
     # Try marc.xml first
     if marc_xml_filename in filenames:
         data = urlopen_keep_trying(item_base + marc_xml_filename).read()
-        if data[:10].find('<?xml') != -1:
+        try:
             root = etree.fromstring(data)
-            return MarcXml(root)
+            if root.tag == "{http://www.loc.gov/MARC21/slim}record":
```

Comment:
Shouldn't this be https ??

Reply:
Good question. I went looking for the answer. Neither http or https for that namespace URI return any resource, but the MARC21slim xsd specifies that URI (with http) as the correct namespace: http://www.loc.gov/standards/marcxml/xml/spy/spy.html  , so it is correct according to the spec.

Reply:
The LoC is not exactly forging ahead with httpseverywhere. https://www.loc.gov/standards/marcxml/xml/spy/spy.html works, but the response refers only to urls using http 
On the wild assumption that this could be changed at any time, perhaps it would be more robust to handle both values of root.tag ?